resume-project/src/components/LeftSide.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Career from './Career.js';
+import Experience from './Experience.js';
+
+const LeftSide = () => (
+  <section className="left-side">
+    <Career />
+    <div className="heading">
+      <h2>
+        <i className="fas fa-check-circle"></i>Experience</h2>
+    </div>
+    <Experience
```

Comment:
can you create a variable w/ an array of json w/ the information for the `Experience` prop and map through it instead of rendering them manually?